index.html

```diff
@@ -101,19 +199,623 @@ <h3>
+
+        </section>
+
+      </section>
+
+      <section>
+        <h3>{{TCPSocket/[[readable]]}} attribute (internal)</h3>
+
+        <aside class="example">
+          An application receiving data from a TCP socket will typically use the {{ReadableStream}} like this:
+
+          <pre class="js">
+            const { readable } = await socket.connection;
+
+            const reader = readable.getReader();
+
+            while (true) {
+              const { value, done } = await reader.read();
+              if (done) {
+                // |reader| has been canceled.
+                break;
+              }
+              // Do something with |value|...
+            }
+
+            reader.releaseLock();
+          </pre>
+        </aside>
+      
+        The steps to <dfn data-lt="initialize tcp readable stream">initialize the TCP readable stream</dfn> are:
+
+        <ol>
+          <li>Let |stream:ReadableStream| be a [=new=] {{ReadableStream}}.
+          <li>
+            Let |pullAlgorithm| be the following steps [=in parallel=]:
+            <ol>
+              <li>
+                Invoke the operating system to read a tcp packet from the socket, placing the result in the [=byte sequence=] |bytes|.
+              <li>
+                [=Queue a global task=] on the [=relevant global object=] of
+                [=this=] using the [=TCP socket task source=] to run the
+                following steps:
+                <ol>
+                  <li>
+                    If EOF packet was received, run the following steps:
+                    <ol>
+                      <li>
+                        Invoke [=ReadableStream/close=] on [=this=].{{TCPSocket/[[readable]]}}.
+                      <li>
+                        Invoke the steps to [=handle closing the readable stream=].
+                    </ol>
+                  <li>
+                    If no errors were encountered run the following steps:
+                    <ol>
+                      <li>
+                        Let |buffer| be a [=new=] {{ArrayBuffer}} created from |bytes|.
+                      <li>
+                        Let |chunk| be a [=new=] {{Uint8Array}} view over |buffer|,
+                        who's length is the length of |bytes|.
+                      <li>
+                        Invoke [=ReadableStream/enqueue=] on
+                        [=this=].{{TCPSocket/[[readable]]}} with |chunk|.
+                    </ol>
+                  <li>
+                    If an operating system error was encountered, invoke
+                    [=ReadableStream/error=] on [=this=].{{TCPSocket/[[readable]]}} with
+                    a "{{NetworkError}}" {{DOMException}} and invoke the steps
+                    to [=handle closing the readable stream=].
+                  <li>
+                    If |the socket was disconnected|, run the following steps:
+                    <ol>
+                      <li>Reject [=this=].{{TCPSocket/[[closedPromise]]}}.
```

Comment:
```suggestion
                      <li>[=Reject=] [=this=].{{TCPSocket/[[closedPromise]]}}.
```

Also, reject with what? Probably the same "NetworkError" DOMException.

Reply:
Fixed!